mmv1/products/orgpolicy/Policy.yaml

```diff
@@ -0,0 +1,204 @@
+# Copyright 2023 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the License);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: Policy
+base_url: '{{parent}}/policies'
+create_url: '{{parent}}/policies'
+self_link: '{{parent}}/policies/{{name}}'
+import_format:
+  ['{{%parent}}/policies/{{name}}']
+update_verb: :PATCH
+references: !ruby/object:Api::Resource::ReferenceLinks
+  guides:
+    'Official Documentation': 'https://cloud.google.com/resource-manager/docs/organization-policy/creating-managing-custom-constraints'
+    'Supported Services': 'https://cloud.google.com/resource-manager/docs/organization-policy/custom-constraint-supported-services'
+  api: 'https://cloud.google.com/resource-manager/docs/reference/orgpolicy/rest/v2/organizations.policies'
+description: "Defines an organization policy which is used to specify constraints for configurations of Google Cloud resources."
+error_retry_predicates: ['transport_tpg.IsOrgpolicyRetryableError']
+examples:
```

Comment:
We're losing the examples in the docs for the more complicated resources. Please copy over at least one of the existing examples